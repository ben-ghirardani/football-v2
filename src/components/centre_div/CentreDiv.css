.centre-div {
  background-color: white;
  border-left: 1px solid black;
  border-right: 1px solid black;
  grid-column: 2;
  display: grid;
  /* can I split text-align? left for text, center for numbers? This would be likely be on table */
  text-align: left;
}