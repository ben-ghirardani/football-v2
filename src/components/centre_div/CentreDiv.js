import React, { Component } from 'react';
import Table from '../table/Table.js'
import Matches from '../matches/Matches.js';
import { Route} from 'react-router-dom';
import './CentreDiv.css'

// does CentreDiv need to be its own component? Can this styling be part of main page?

export default class CentreDiv extends Component {

  render() {
    return(
      <div className="centre-div">
        {/* <Router> */}
          <>
            <Route path="/" exact render=
              {() => 
                <Table
                  standings={this.props.standings}
                  updateCurrentTeam={this.props.updateCurrentTeam} 
                  getTeamStoredMatches={this.props.getTeamStoredMatches}
                  currentTeam={this.props.currentTeam}
                  currentTeamID={this.props.currentTeamID}
                  matches={this.props.matches}
                  updateTeamMatches={this.props.updateTeamMatches}
                />
              }
            />
            <Route path="/fixtures/" render=
              {() => 
                <Matches
                  currentTeam={this.props.currentTeam} 
                  currentTeamID={this.props.currentTeamID}
                  matches={this.props.matches}
                  accessStoredTeamMatches={this.props.accessStoredTeamMatches}
                />
              }
            />
          </>
        {/* </Router> */}
      </div>
    )
  }

}