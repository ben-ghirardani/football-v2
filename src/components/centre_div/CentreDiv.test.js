import React from 'react';
import CentreDiv from './CentreDiv';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
configure({ adapter: new Adapter() });

it('CentreDiv renders as expected', ()=> {
  const wrapper = shallow(<CentreDiv  />);
  expect(wrapper).toMatchSnapshot();
})