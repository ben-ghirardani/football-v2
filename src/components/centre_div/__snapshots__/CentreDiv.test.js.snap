// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CentreDiv renders as expected 1`] = `ShallowWrapper {}`;
